ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -31,8 +36,9 @@ cat << EOF > ~/ocp-install.yml
     retries: 5
   - name: Remove last run
     shell: kcli delete plan --yes upstream_ci
+    ignore_errors: yes
```

Comment:
+1 on this
but seems unrelated to this commit.
it is better for git housekeeping in and clean history if this is a separate commit (can still go in same PR)

Reply:
Fixed